src/toaster/src/Toaster.js

```diff
@@ -15,15 +16,29 @@ export default class Toaster {
     container.setAttribute('data-evergreen-toaster-container', '')
     document.body.appendChild(container)
 
-    ReactDOM.render(
+    const el = () => (
```

Comment:
Can we name this `toastManager`? (Additionally, I don't think it needs to be wrapped in a function since both `createRoot(container).render()` and `render()` accept the component instance)

Reply:
I wrapped it inside a function because this way we don't have to repeat the same code of the component inside the `.render()` function because it will always be the same

Thanks for the feedback @brandongregoryscott